java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -282,4 +282,35 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<ClusterValue<String[]>> time(Route route);
+
+    /**
+     * Synchronously saves the DataBase.<br>
```

Comment:
Is it better to copy the documentation on Valkey. Seems a bit more detailed and concise enough?
Performs a synchronous save of the dataset producing a point in time snapshot of all the data inside the Valkey instance, in the form of an RDB file.

Reply:
But server may use AOF, not RDB...
I mostly aligned this doc with `bgsave`, rather than with redis/valkey doc

Reply:
But I'm ready to update it

Reply:
https://redis.io/docs/latest/commands/save/ can it doesn't talk about it in the documentation for AOF. it was mostly a nit comment. Seemed a bit too concise. You can resolve if you think its not necessary.

Reply:
I think you should mention "snapshot of the database" in your documentation.  
Why DataBase vs database?